### PR TITLE
[cli] Clean up TerminalUI

### PR DESCRIPTION
**Pull request by EvanBacon**
- make the terminal UI shorter
- group the commands 
- match up wording across commands 
- standardize text formatting

currently

### Before

Screen Shot 2020-09-09 at 10 07 54 AM

### After

Screen Shot 2020-09-09 at 10 24 10 AM

### CODE REVIEW

File: packages/expo-cli/src/commands/start/TerminalUI.ts [hunk 1]
```diff
@@ -45,37 +45,56 @@ const printHelp = (): void => {
...
-`);
+  const currentAuth = `@${username}`;
+  const currentToggle = openDevToolsAtStartup ? 'disabled' : 'enabled';
+
+  const isMac = process.platform === 'darwin';
+
+  const ui = [
+    [],
+    ['a', `open Android`],
+    ['shift+a', `select a device or emulator`],
```
Comment (fson) [inline packages/expo-cli/src/commands/start/TerminalUI.ts:63]: I wonder if we could use something like `⇧+a` here? Although I guess some fonts might not have this character.

**Review by fson — APPROVED** [review]
💯❤️

File: packages/expo-cli/src/commands/start/TerminalUI.ts (hunk 1 above)
Comment (EvanBacon) [reply packages/expo-cli/src/commands/start/TerminalUI.ts:63]: I tried that but it was too small in the default terminal :(

**Review by brentvatne — APPROVED** [review]
pretty good

**Pull request merged**